.travis.yml

```diff
@@ -5,10 +5,13 @@ addons:
   apt:
     sources: 
       - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty-5.0
+      - sourceline: 'deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-5.0 main'
+        key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
```

Comment:
As an alternative to the llvm snapshots, I've used the following before....

```
addons:
  apt:
    sources:
    - sourceline: 'ppa:ubuntu-toolchain-r/test'
    packages:
    - cmake
    - time
    - g++-7
    - gcc-7
    - binutils
    - libc6
    - libc6-amd64
    - libc6-dev
    - libc6-dbg
```

Reply:
Travis pipeline for OSX uses XCode, which as far as I know has its own version of Clang, so I don't think this will help much.